Java Migration/README.md

```diff
@@ -0,0 +1,27 @@
+## Steps for migrating to Azul Zulu OpenJDK from Oracle JRE
+
+Search feature of Azure DevOps Server uses Elasticsearch which depends on Java Runtime Environment (JRE). Up until now, Oracle JRE was the supported version of JRE.  Given the updates to Oracle Java licensing terms which now requires one to buy a subscription to continue to get updates to JRE for commercial use, Azure DevOps Server Search will support Azul Zulu OpenJDK so that our users have an option to choose between Azul Zulu OpenJDK and Oracle JRE, a decision they can make based on their business needs.
```

Comment:
As mentioned, the rationale behind moving to zulu is the free updates. But in the last line, it doesn't convey clearly how would a move to zulu, help in free updates. Would we auto update zulu java or customer has to update. It looks ambiguous to me.

Reply:
fixed it